### PR TITLE
Helm Collector

### PR DESCRIPTION
**Pull request by codeprc**
Change Release Details : Helm Collector

A collector to showcase the Kubernetes Deployment Container details
https://helm.sh/docs/intro/using_helm/https://helm.sh/docs/intro/quickstart/

Description:
The Helm tool is an standard tool for Kubernestes DeploymentThis Helm 1.3 client is a client side tool and direclty talks to kubernetes with kube config fileThis collector collects the deployment details of Releases, Charts & RepoIt simply fires the commandline command and collects/updates the Data in MongoThe following commands it fires

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-@Hygieia/hygieia-iac-terraform-collector
+@Hygieia/hygieia-helm-terraform-collector
```
Comment (rvema) [inline .github/CODEOWNERS]: the github admin team for this collector is hygieia-helm-collector, please change it to `@Hygieia/hygieia-helm-collector`

File: src/changes/changes.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+mvn changes:announcement-generate -Prelease-notes [-Dchanges.version=nnn]
+then tweak the formatting if necessary
+and commit
+The <action> type attribute can be add,update,fix,remove.
+-->
+
+
+<document>
+    <properties>
+        <title>Hygieia Docker Collector Changes</title>
```
Comment (rvema) [inline src/changes/changes.xml]: please change the title to Helm Collector

File: src/main/java/com/capitalone/dashboard/model/Release.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Transient;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import com.capitalone.dashboard.enums.HelmStatus;
+
+/*Model to store the latest release Data*/
+
+@Document(collection="helm_release")
```
Comment (rvema) [inline src/main/java/com/capitalone/dashboard/model/Release.java:15]: Is there a way we can work with our current release collection? We feel the data model should not be tool dependent. Let me know what you feel

File: src/main/java/com/capitalone/dashboard/model/Repo.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
+package com.capitalone.dashboard.model;
+
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "helm_repo")
```
Comment (rvema) [inline src/main/java/com/capitalone/dashboard/model/Repo.java:6]: we need to come up with data model that is tool agnostic

**Review by rvema — CHANGES_REQUESTED** [review]
Please address the review comments and also please setup some time when we can discuss more about the datamodel. Additionally please copy the `.github` folder from [hygieia-whitesource-collector](https://github.com/Hygieia/hygieia-whitesource-collector) project to the root of this project. Additionally please address the snyk vulnerabilities raised as part of PRs #2 , #3 and #4 as part of this PR.

**Comment by rvema** [comment]
@prash897 please address the above review comments.

File: src/main/java/com/capitalone/dashboard/model/Repo.java (hunk 4 above)
Comment (codeprc) [reply src/main/java/com/capitalone/dashboard/model/Repo.java:6]: Resolved

**Comment by codeprc** [comment]
Changes Pushed

**Review by rvema — CHANGES_REQUESTED** [review]
can you include the changes proposed as part of automated PRs #2 #3 and #4 in this pull request? Additionally there seems to be an issue with Readme file, can you remove it and commit it later? I am unable to merge the PR due to this conflict. please let me know if you need additional information in this regard.

**Review by rvema — APPROVED** [review]
lgtm

**Pull request merged**